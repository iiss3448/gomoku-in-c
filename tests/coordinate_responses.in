place A1
place A2B2
place A20
place A1
place A01
place AB1
place A111
place A0
plac B1
1121089799101 C10
place A2 
place A2
term
