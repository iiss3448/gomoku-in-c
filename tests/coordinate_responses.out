Invalid coordinate
Invalid coordinate
Occupied coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid!
Invalid!
Invalid!
